### PR TITLE
chore: add atlas-local to simplify local dev against mms

### PR DESCRIPTION
**Pull request by mcasimir**
`ELECTRON_EXTRA_ARGS="--atlasServiceBackendPreset=atlas-local" npm run start`

### CODE REVIEW

File: packages/compass/src/main/application.ts [hunk 1]
```diff
@@ -145,6 +145,16 @@ class CompassApplication {
         },
         authPortalUrl: 'https://account-dev.mongodb.com/account/login',
       },
+      'atlas-local': {
+        atlasApiBaseUrl: 'http://localhost:8080',
+        atlasApiUnauthBaseUrl:
+          'https://cloud-dev.mongodb.com/api/private/unauth',
```
Comment (Anemy) [inline packages/compass/src/main/application.ts:151]: Should this also be local host?
```suggestion
          'http://localhost:8080/api/private/unauth',
```
I can give it a manual test on local if it helps to confirm.

**Review by Anemy — COMMENTED** [review]
Added questions on pathing and hello endpoint that we might want to confirm with a manual local test.

File: packages/compass/src/main/application.ts (hunk 1 above)
Comment (mcasimir) [reply packages/compass/src/main/application.ts]: that would be awesome!

**Review by Anemy — APPROVED** [review]
@mcasimir Ran it against local mms. Looks good 👍

**Pull request merged**